rt/rs/microprofile-client/src/main/java/org/apache/cxf/microprofile/client/JsonPProvider.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.cxf.microprofile.client;
+
+import javax.annotation.Priority;
+import org.apache.cxf.jaxrs.provider.jsrjsonp.JsrJsonpProvider;
+
+@Priority(Integer.MAX_VALUE)
+public class JsonPProvider extends JsrJsonpProvider {
```

Comment:
I believe I can now get rid of this, and instead manually add a ProviderInfo with custom = false.